### PR TITLE
v3.4.0

### PR DESCRIPTION
**Pull request by webNeat**
## Testing

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,13 @@
 
 ---
 
+## v3.4.0
+
+- Migrate `serializeAttribute()` from privacy.ts to serialize.ts ([#1033](https://github.com/DataDog/browser-sdk/pull/1033))
+- Remove monitoring & ignore untrusted events ([#1029](https://github.com/DataDog/browser-sdk/pull/1029))
+- v3.3.1 ([#1031](https://github.com/DataDog/browser-sdk/pull/1031))
```
Comment (bcaudan) [inline CHANGELOG.md:23]: no need to include previous release PR in the changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,13 @@
 
 ---
 
+## v3.4.0
+
+- Migrate `serializeAttribute()` from privacy.ts to serialize.ts ([#1033](https://github.com/DataDog/browser-sdk/pull/1033))
```
Comment (bcaudan) [inline CHANGELOG.md:21]: probably not interesting for customers

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,6 +16,13 @@
...
+- Remove monitoring & ignore untrusted events ([#1029](https://github.com/DataDog/browser-sdk/pull/1029))
```
Comment (bcaudan) [inline CHANGELOG.md:22]:
```suggestion
- ForegroundContext: Remove monitoring & ignore untrusted events ([#1029](https://github.com/DataDog/browser-sdk/pull/1029))
```

File: CHANGELOG.md (hunk 1 above)
Comment (webNeat) [reply CHANGELOG.md]: I updated the commit

Comment (bcaudan) [reply CHANGELOG.md]: cf release doc, you'll have to update the tag as well

Comment (webNeat) [reply CHANGELOG.md]: Done

**Review by bcaudan — APPROVED** [review]

**Pull request merged**